components/usage/pkg/db/workspace_instance_test.go

```diff
@@ -277,13 +288,7 @@ func TestListWorkspaceInstancesInRange(t *testing.T) {
 
 	dbtest.CreateWorkspaceInstances(t, conn, all...)
 
-	retrieved, err := db.ListWorkspaceInstancesInRange(context.Background(), conn, startOfMay, startOfJune)
-	require.NoError(t, err)
-
-	ids := []uuid.UUID{}
-	for _, ws := range retrieved {
-		ids = append(ids, ws.ID)
-	}
+	retrieved := dbtest.ListWorkspaceInstancesInRange(t, conn, startOfMay, startOfJune, workspace.ID)
```

Comment:
The listing methods should not be in the `dbtest` package, should be in the `db` package instead.

Reply:
It is a testing utility that requires to provide a workspaceId and filters the result on it to make tests more stable. Should I name it differently to make it clearer?